### PR TITLE
Merge master into dev for packaging

### PR DESCRIPTION
**Pull request by pvanderknyff**
Bring the packaging changes into dev.

### CODE REVIEW

File: samples/plugins/postgres_odbc/connectionBuilder.js [hunk 1]
```diff
@@ -11,7 +11,12 @@
...
-
+  
+    if (attr[connectionHelper.attributeSSLMode] == "require")
+    {        
+        params["sslmode"] = "require";
+    }
+  
     var odbcConnectStringExtrasMap = {};
     const attributeODBCConnectStringExtras = "odbc-connect-string-extras";
     if (attributeODBCConnectStringExtras in attr)
```
Comment (lriggs) [inline samples/plugins/postgres_odbc/connectionBuilder.js]: Isn't this a 2020.1 feature?

Comment (pvanderknyff) [reply samples/plugins/postgres_odbc/connectionBuilder.js]: This is moving master into dev, not dev into master. This will not affect master.

Comment (pvanderknyff) [reply samples/plugins/postgres_odbc/connectionBuilder.js]: Except the resolve commit merged dev into master...? Yep, that's not what I expected to happen. I'll fix it.

**Comment by pvanderknyff** [comment]
Messed up the pull request, will try again

**Pull request closed without merging**